### PR TITLE
add v9 version of examine quick start

### PR DESCRIPTION
**Pull request by mrexitium**
Haven't improved on the guide, just made it work for v9 - probably lots of things that could be improved upon 😊

### CODE REVIEW

**Comment by eshanrnh** [comment]
Thanks for the PR, @mrexitium 💪

We will review it asap 🙂

File: Reference/Searching/Examine/Quick-Start/index.md [hunk 1]
```diff
@@ -159,71 +174,75 @@ The final template looks like this:
...
                         }
                         <div class="employee-grid__item__contact">
-                            @SocialLink(person.FacebookUsername, "Facebook")
-                            @SocialLink(person.TwitterUsername, "Twitter")
-                            @SocialLink(person.LinkedInUsername, "LinkedIn")
-                            @SocialLink(person.InstagramUsername, "Instagram")
+                            @{ SocialLink(person.FacebookUsername, "Facebook"); }
+                            @{ SocialLink(person.TwitterUsername, "Twitter"); }
+                            @{ SocialLink(person.LinkedInUsername, "LinkedIn"); }
+                            @{ SocialLink(person.InstagramUsername, "Instagram"); }
```
Comment (dampee) [inline Reference/Searching/Examine/Quick-Start/index.md:238]: Are you sure about this?

I don't see where the SocialLink() is coming from.

Comment (jemayn) [reply Reference/Searching/Examine/Quick-Start/index.md:238]: Sorry I missed your review here Damiaan!

I haven't touched this for any Examine related reasons. 

The SocialLink() method is part of [the starter kit](https://github.com/umbraco/The-Starter-Kit/blob/v3.0.0/src/Umbraco.SampleSite.Website/Views/People.cshtml#L42-L45) which this code is extending, don't remember why I had to add this razor code around them, but if it works without we can just remove it - I seem to remember running into some issues with it throwing errors though.

**Comment by sofietoft** [comment]
Hi @mrexitium !

Since this PR was made, another PR with similar changes has been merged: #4143 .

I'll close this PR for now.
Please let me know if you have any questions or comments 🙌 

Thanks again for helping us keep the docs up to date!

**Pull request closed without merging**